### PR TITLE
feat(integrations): add support for productboard

### PR DESCRIPTION
**Pull request by hassan254-prog**
## Describe your changes

- Added configuration for productboard in providers.yaml
- Added productboard to docs-v2.
- Added productboard pages for Supported API groups in mint.json.
- Added productboard SVG file in template-logos folder.
- Consolidate productboard docs pages

## Issue ticket number and link
[EXT-70](https://linear.app/nango/issue/EXT-70/add-productboard-api)
## Test
This provider has been used successfully in local development to connect to productboard after authorizing using Nango.
The documentation update was reviewed using `npm run docs` and verifying on localhost.

### CODE REVIEW

File: docs-v2/integrations/all/productboard.mdx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+| [Sync data](/integrate/guides/sync-data-from-an-api) | ✅ |
+| [Perform workflows](/integrate/guides/perform-workflows-with-an-api) | ✅ |
+| [Proxy requests](/integrate/guides/proxy-requests-to-an-api) | ✅ |
+| [Receive webhooks](/integrate/guides/receive-webhooks-from-an-api) | 🚫 (time to contribute: &lt;48h) |
+
+<Tip>We can implement missing features in &lt;48h, just ask for it in the [community](https://nango.dev/slack).</Tip>
+
+## Getting started
+
+-   [How Register an Application](https://developer.productboard.com/docs/how-to-integrate-with-productboard-via-oauth2-developer-documentation#registering-an-oauth2-application)
```
Comment (bodinsamuel) [inline docs-v2/integrations/all/productboard.mdx]:
```suggestion
-   [How to Register an Application](https://developer.productboard.com/docs/how-to-integrate-with-productboard-via-oauth2-developer-documentation#registering-an-oauth2-application)
```

File: docs-v2/integrations/all/productboard.mdx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+-   [OAuth-related docs](https://developer.productboard.com/docs/how-to-integrate-with-productboard-via-oauth2-developer-documentation)
```
Comment (bodinsamuel) [inline docs-v2/integrations/all/productboard.mdx]:
```suggestion
-   [OAuth related docs](https://developer.productboard.com/docs/how-to-integrate-with-productboard-via-oauth2-developer-documentation)
```

File: docs-v2/integrations/all/productboard.mdx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+-   [Scopes](https://developer.productboard.com/docs/how-to-integrate-with-productboard-via-oauth2-developer-documentation#access-scopes)
+-   [Productboard API docs (their REST API)](https://developer.productboard.com/reference/introduction)
```
Comment (bodinsamuel) [inline docs-v2/integrations/all/productboard.mdx]:
```suggestion
-   [Productboard REST API docs](https://developer.productboard.com/reference/introduction)
```

**Review by bodinsamuel — COMMENTED** [review]
📈

**Comment by hassan254-prog** [comment]
@bodinsamuel thank you for catching that 🙏

**Review by bodinsamuel — APPROVED** [review]

**Review by khaliqgant — APPROVED** [review]

File: packages/shared/providers.yaml [hunk 4]
```diff
@@ -1727,6 +1727,24 @@ posthog:
...
+    scope_seprator: ' '
+    authorization_params:
+        response_type: code
+    token_params:
+        grant_type: authorization_code
+    refresh_params:
+        grant_type: refresh_token
+    proxy:
+        headers:
+            X-Version: 1
```
Comment (khaliqgant) [inline packages/shared/providers.yaml]: Can we make this a "1" (string) so yaml validation passes?

**Review by khaliqgant — CHANGES_REQUESTED** [review]
Small configuration change requested

File: packages/shared/providers.yaml (hunk 4 above)
Comment (hassan254-prog) [reply packages/shared/providers.yaml]: @khaliqgant, thanks for catching that. 🙏

**Review by bodinsamuel — APPROVED** [review]

**Pull request merged**